config/crd/kustomization.yaml

```diff
@@ -3,17 +3,20 @@
 # It should be run by config/default
 resources:
 - bases/core.openfeature.dev_featureflagconfigurations.yaml
+- bases/core.openfeature.dev_flagdconfigurations.yaml
 #+kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
 - patches/webhook_in_featureflagconfigurations.yaml
+#- patches/webhook_in_flagdconfigurations.yaml
```

Comment:
why are these disabled unlike the featureflagconfigurations webhook?

Reply:
I have enabled them, missed this on my first pass